### PR TITLE
Milestone 1: Set up GitHub and tools

### PR DESCRIPTION
**Pull request by Emmyn5600**
Hey!

This pull request contains the  following changes:

- Configured Linters and tools
- Added a README file
- Added index.html file
- Added style.css file
- Added Header, with logo, search, login, and subscribe button.
- Added a column with the main content
- Added content in the middle, bottom, and footer.
- Added footer page
- Fixed linter errors
- Updated README file
- Worked on feedbacks

Regards,
Emmy N.

### CODE REVIEW

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Invalid PR 😞 
Hey @Emmyn5600 I appreciate the efforts you put into this project. Keep up!
However, I can not continue with the review because you have failing linter cases and therefore this is an invalid PR.
Please fix the failing linter cases and resubmit for another review.


### Martin Afani

File: index.html [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    <link  rel="shortcut icon" href="/vi-assets/static-assets/favicon-d2483f10ef688e6f89e23806b9700298.ico">
+    <link rel="stylesheet" href="./assets/style.css">
+    <script src="https://kit.fontawesome.com/fd0eaf46fd.js" crossorigin="anonymous"></script>
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times</title>
+  </head>
+  <body>
+  <main>
+      <header>
+          <section class="head_position">
+            <nav id="Head_logo" class="flex">
```
Comment (wathigo) [inline index.html:17]: Let's improve styles on the nav section of the page:
* The icons have not been styled well. They should be centered inside a circular border.
* The search and hamburger icons are too small. 
* Space and Cosmos title should occupy one line.
* The text inside the login and subscribe buttons needs some margin on the left and right.

Clone Page | Original Page
|---------------:|:------------------|
Screenshot from 2021-02-18 19-03-58 | original_nav

File: index.html [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+         <p class="content_2_paragraph">Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+     </div>
+     <div class="inflation_info">
+        <h1 class="inflation_title">The Theory of Inflation</h1>
+        <p class="inflation_summary">
+            <span>Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</span>
+        </p>
+        <div class="inflation_image">
+            <div class="inflation_image_position">
+                <img src="./assets/cause1.png" alt=""></div>
```
Comment (wathigo) [inline index.html:118]: The 'Theory of Inflation' section is quite different from the original page.
Have a look. Let's work on the styling.
Cloned Version | Original Version
|-------------------:|:----------------------|
Screenshot from 2021-02-18 19-12-40 | original_inflation

File: index.html [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+        </div>
+        <hr class="small_line_2"> 
+        <section class="body_content">
+         <div class="body_content_head">
+            <strong><span class="section_2_title"><a href="#">More in Space and Astronomy</a></span></strong>
+         </div>
+         <div class="body_content_image">
+             <div class="body_content_image_1">
+                <img src="./assets/NASA.jpg" alt="">
+                <section class="content_image_caption">NASA</section>
```
Comment (wathigo) [inline index.html]: The bottom part is also completely different from what we have in the original version. Let's check out this :point_down: and make the necessary changes.
Cloned Version | Original Version
|--------------------:|:---------------------|
Screenshot from 2021-02-18 19-22-07 | original_bottom

File: index.html [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+            <a href="#">Newsletter</a>
+            <a href="#">TimesMachine</a>
+            <a href="#">NYT Store</a>
+            <a href="#">Times Journeys</a>
+            <a href="#">Manage My Account</a>
+        </div>
+    </nav>
+    
+</div>
+<footer class="footer">
```
Comment (wathigo) [inline index.html:489]: Lastly, we have a column missing from the footer.
Cloned Version | Original Version
---------------------:|:--------------------|
Screenshot from 2021-02-18 19-27-18 | original_footer

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job on this project.
Let's address a few issues before we proceed. See the comments :point_down: 

**After making the necessary changes, submit another request for review.** 

**Happy Debugging :technologist: 
Simon Wathigo**

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required 🛑

### Hello team 👋,

**Great job implementing the requested changes 👏.**

### Highlights

- **The page looks like the original.**
- **The code is clean.**

### Mandatory Requirements.

1). Improve the following sections so they look like the original page.

- **space & cosmos text in the navbar should appear on a single line.**
- **For the theory of inflation section, there should be some spacing on the left and right. Also, increase the size of the images.**
- **Fix the styling of the last column in the Footer section.**
- **Fix the alignment of the date captions in More in Space and Anatomy and Editor's pick sections.**
- 

2). **The new york times logo in the footer should be a link.**

### Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

**Review by ghost — APPROVED** [review]
# Status: Approved 🎉 

**Congratulations team 👏. Good luck with your next project 😃.**

**Pull request merged**